pulsar-functions/proto/src/main/proto/InstanceCommunication.proto

```diff
@@ -49,8 +48,6 @@ message FunctionStatus {
     // expressed in ms since epoch
     int64 lastInvocationTime = 13;
     string instanceId = 14;
-//    MetricsData metrics = 15 [deprecated=true];
```

Comment:
lets leave the commented out defs so that we know which indices have been used in the past for backwards compatibility reasons